en/docs/develop-components/develop-proxy/develop-an-api-proxy.md

```diff
@@ -27,29 +27,35 @@ To create an API proxy, you can either upload an OpenAPI specification or provid
 1. Go to [https://console.choreo.dev/](https://console.choreo.dev/) and sign in. This opens the project home page.
 2. If you already have one or more components in your project, click **+ Create**. Otherwise, proceed to the next step.
 3. Click the **API Proxy** card. This opens the **Create an API Proxy** pane.
-4. Click **Skip Source** to define resources manually.
-5. Specify the following values as API proxy details:
+4. Enter the following details:
 
     !!! info
-        The **Name** field must be unique and cannot be changed after creation.
+        The **Component Name** field must be unique and cannot be changed after creation.
+
+    | **Field**                 | **Value**          |
+    |---------------------------|--------------------|
+    | **Component Display Name**| `Department Service`          |
+    | **Component Name**        | `departmentService`          |
+    | **Description**           | `Sample proxy for department service`     |
+
+5. Click **Create from scratch** tab to define resources manually.
+6. Specify the following values as API proxy details:
 
     | **Field**       | **Value**                                  |
     |-----------------|--------------------------------------------|
-    | **Display Name**| `HR API`                                   |
-    | **Name**        | `hr-api`                                   |
-    | **Context**     | `abc-hr`                                   |
+    | **Context**     | `department/v1`                                   |
     | **Version**     | `1.0`                                      |
-    | **Target**      | `https://samples.choreoapps.dev/company/hr`|
+    | **Target**      | `https://samples.choreoapps.dev/company/hr/department`|
```

Comment:
Do we need to change this?
Wouldn't keeping Target as "https://samples.choreoapps.dev/company/hr" better than adding the /department part into it, which causes a `{departmentId}` resource which looks a bit odd?